README.md

```diff
@@ -55,6 +55,9 @@ java -Dport=8081 --module-path=server/target/compiler/ \
   -jar server/target/mod-reservoir-server-fat.jar
 ```
 
+Note: Prometheus metrics are reported on port `8082` and path `/metrics` by default.
+The port can be change by passing the `-Dprometheus.port=` option.
+
```

Comment:
I don't think it should be enabled by default. I think when you specify port, then it's enabled.